PlayMaker/PlayMaker/PlayMakerInfrastructure/PlayMakerDbSeeder.cs

```diff
@@ -0,0 +1,177 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using PlayMakerDomain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PlayMakerInfrastructure
+{
+    public class PlayMakerDbSeeder
+    {
+        private readonly PlayMakerDbContext _dbContext;
+
+        public PlayMakerDbSeeder(PlayMakerDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public void Seed()
+        {
+            if (_dbContext.Database.CanConnect())
+            {
+                if (!_dbContext.Roles.Any())
+                {
+                    var roles = CreateRoles();
+                    _dbContext.Roles.AddRange(roles);
+                    _dbContext.SaveChanges();
+                }
+
+                if (!_dbContext.SportType.Any())
+                {
+                    var types = CreateSportTypes();
+                    _dbContext.SportType.AddRange(types);
+                    _dbContext.SaveChanges();
+                }
+
+                if (!_dbContext.Game.Any())
+                {
+                    var games = CreaateGames();
+                    _dbContext.Game.AddRange(games);
+                    _dbContext.SaveChanges();
+                }
+                if (!_dbContext.Payment.Any())
+                {
+                    var payments = CreatePayments();
+                    _dbContext.Payment.AddRange(payments);
+                    _dbContext.SaveChanges();
+                }
+            }
+        }
+
+        private List<IdentityRole> CreateRoles()
+        {
+            var roles = new List<IdentityRole>()
+            {
+                  new IdentityRole()
+                  {
+                      Name = "admin",
+                      NormalizedName = "admin"
+                  },
```

Comment:
Please add rolse and change rest of them to have hierarchy like this:
1. Admin - all priviliges everywhere
2. Developer - lot of priviliges
3. PremiumUser eg user who is handling lot of games and is trusted
4. VerifiedUser eg user who confirmed email etc
5. User - standard

Reply:
I removed roles table and identity nuget package since it will be done in distinct branch